Platon/backend/tests/milestone_test.py

```diff
@@ -0,0 +1,229 @@
+## ASSUMPTION!!!!!!!!
+        # workspace state is 1, for Search for Collaborators State
+        # workspace state is 2, for Ongoing State
+        # workspace state is 3, for Published State
```

Comment:
As far as I know, you and @alperendivriklioglu do not use the same number pattern to represent states. Please create an enum to represent them.

Reply:
I've updated for both milestoneAPI and issuesAPI. Now states start from 0, same as @alperendivriklioglu implemented. Also, all unit tests of milestoneAPI and issueAPI are passed after these changes